src/OrleansAzureUtils/Providers/Storage/AzureTableStorage.cs

```diff
@@ -53,8 +54,12 @@ public class AzureTableStorage : IStorageProvider, IRestExceptionDecoder
         private bool isDeleteStateOnClear;
         private static int counter;
         private readonly int id;
-        private const int MAX_DATA_SIZE = 64 * 1024; // 64KB
+        private const int MAX_DATA_CHUNK_SIZE = 64 * 1024; // 64KB
+        private const int MAX_DATA_CHUNKS_COUNT = 15;
```

Comment:
As @gabikliot mentioned above, please add comment(s) into the code to explain these calculations / numbers so that the rational is preserved for anyone that has to read / change the code in future.